packages/api-client/README.md

```diff
@@ -1,6 +1,6 @@
 # @opendatasoft/api-client ![CI status](https://github.com/opendatasoft/ods-dataviz-sdk/workflows/CI/badge.svg)
 
-This package implements a Typescript/Javascript client library for [Opendatasoft's Search APIv2](https://help.opendatasoft.com/apis/ods-search-v2/#search-api-v2).
+This package implements a Typescript/Javascript client library for [Opendatasoft's Search APIv2.1](https://help.opendatasoft.com/apis/ods-explore-v2/explore_v2.1.html).
```

Comment:
The `README.md` at the root of the project has also a link to the explore api documentation